src/classes/STG_InstallScript.cls

```diff
@@ -284,4 +237,69 @@ global without sharing class STG_InstallScript implements InstallHandler {
             upsert orgSettings;
         }
     }
+
+    /*******************************************************************************************************
+    * @description Schedules a class that implements the Schedulable interface, if it's not already scheduled.
+    * @param JobName The name to give the scheduled job.
+    * @param frequency The frequency to schedule the job at.
+    * @param className The class to schedule. Should implement the Schedulable interface.
+    * @return The ID of the scheduled job.
+    */
+    public static String scheduleJobIfNotScheduled(String JobName, String frequency, String className) {
+
+        //Check if the job is already scheduled - CronJobDetail is the parent and CronTrigger is the child
+        //Type '7' is for scheduled Apex
+        List<CronTrigger> scheduledJobs = [select Id, CronJobDetail.Id from CronTrigger
+                    where CronJobDetail.Name = :JobName and CronJobDetail.JobType = '7'];
+
+        if(scheduledJobs.size() == 0) {
+            Type classType = Type.forName(className);
+
+            if(classType != null) {
+               Object instance = classType.newInstance();
+               if(instance instanceof Schedulable)
```

Comment:
friends don't let friends skip the curly braces..

Reply:
Oh goodness! This is pre-existing code but I'll change it for sure!

Reply:
no problem...it was showing up as all green for me otherwise I'd probably would have just noted.